### PR TITLE
[CONFORMANCE][IE TEESTS] Move Summary from ngraph:: to ov::

### PR DESCRIPTION
**Pull request by iefode**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/tests/functional/plugin/shared/src/behavior/ov_plugin/caching_tests.cpp [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
 #include "behavior/ov_plugin/caching_tests.hpp"
 
+#include "ngraph/pass/manager.hpp"
```
Comment (ilyachur) [inline src/tests/functional/plugin/shared/src/behavior/ov_plugin/caching_tests.cpp:11]: Why do you use old headers?

File: src/tests/ie_test_utils/functional_test_utils/src/summary/op_summary.cpp [hunk 2]
```diff
@@ -28,16 +29,16 @@ void OpSummaryDestroyer::initialize(OpSummary *p) {
...
-    opsets.push_back(ngraph::get_opset2());
-    opsets.push_back(ngraph::get_opset3());
-    opsets.push_back(ngraph::get_opset4());
-    opsets.push_back(ngraph::get_opset5());
-    opsets.push_back(ngraph::get_opset6());
-    opsets.push_back(ngraph::get_opset7());
-    opsets.push_back(ngraph::get_opset8());
-    opsets.push_back(ngraph::get_opset9());
-    opsets.push_back(ngraph::get_opset10());
+    opsets.push_back(ov::get_opset1());
```
Comment (ilyachur) [inline src/tests/ie_test_utils/functional_test_utils/src/summary/op_summary.cpp:32]: Can you use `ov::get_available_opsets()` method?

File: src/tests/ie_test_utils/functional_test_utils/src/summary/op_summary.cpp [hunk 3]
```diff
@@ -135,41 +136,41 @@ std::map<std::string, PassRate> OpSummary::getStatisticFromReport() {
     return oldOpsStat;
 }
 
-void OpSummary::updateOPsStats(const std::shared_ptr<ngraph::Function> &function, const PassRate::Statuses &status) {
-    if (function->get_parameters().empty()) {
+void OpSummary::updateOPsStats(const std::shared_ptr<ov::Model> &Model, const PassRate::Statuses &status) {
```
Comment (ilyachur) [inline src/tests/ie_test_utils/functional_test_utils/src/summary/op_summary.cpp:139]:
```suggestion
void OpSummary::updateOPsStats(const std::shared_ptr<ov::Model> &model, const PassRate::Statuses &status) {
```

File: src/tests/ie_test_utils/functional_test_utils/src/summary/op_summary.cpp [hunk 4]
```diff
@@ -180,33 +181,33 @@ void OpSummary::updateOPsStats(const std::shared_ptr<ngraph::Function> &function
     }
 }
 
-void OpSummary::updateOPsImplStatus(const std::shared_ptr<ngraph::Function> &function, const bool implStatus) {
-    if (function->get_parameters().empty()) {
+void OpSummary::updateOPsImplStatus(const std::shared_ptr<ov::Model> &Model, const bool implStatus) {
```
Comment (ilyachur) [inline src/tests/ie_test_utils/functional_test_utils/src/summary/op_summary.cpp:184]: the same

File: src/tests/ie_test_utils/functional_test_utils/src/summary/op_summary.cpp (hunk 2 above)
Comment (iefode) [reply src/tests/ie_test_utils/functional_test_utils/src/summary/op_summary.cpp]: Fixed :) I used this API in other places, here i decided to take only version

File: src/tests/functional/plugin/shared/src/behavior/ov_plugin/caching_tests.cpp (hunk 1 above)
Comment (iefode) [reply src/tests/functional/plugin/shared/src/behavior/ov_plugin/caching_tests.cpp]: Fixed

File: src/tests/ie_test_utils/functional_test_utils/src/summary/op_summary.cpp (hunk 3 above)
Comment (iefode) [reply src/tests/ie_test_utils/functional_test_utils/src/summary/op_summary.cpp]: Fixed :) Blind replacement

**Review by ilyachur — APPROVED** [review]

**Pull request merged**